package.json

```diff
@@ -23,33 +23,26 @@
   ],
   "module": "lib/main.js",
   "dependencies": {
+    "@digitalbazaar/minimal-jwt": "^1.0.1",
+    "auth-header": "^1.0.0",
+    "base64url-universal": "^1.1.0",
+    "bedrock": "^4.1.1",
+    "bedrock-https-agent": "^2.0.0",
+    "bedrock-mongodb": "^8.1.1",
+    "bedrock-tokenizer": "^1.1.0",
```

Comment:
`bedrock` modules will be peer deps here.

Reply:
Removed all bedrock stuff since we won't be using it in this library. Marking this and other comments related to bedrock stuff resolved.